docs/_docs/quick-start/getting-started.md

```diff
@@ -25,54 +24,53 @@ apm install nuclide
+After installation, launch Nuclide by [opening Atom](/docs/editor/basics/#opening). Once Atom
+is open, you should see the Nuclide Home page.
 
 ![](/static/images/docs/quick-start-getting-started-home-page.png)
 
-The left-hand side pane is the Nuclide file-tree view. The main pane contains introductory
-information about Nuclide and the Quick Launch Menu. This main pane is also where you will be
-editing your files (just like in normal Atom). And there is a bottom status bar to show you error
-and health statistics.
+- The left side-pane is the Nuclide File Tree view.
+- The main pane contains introductory information about Nuclide and the Quick Launch Menu. This is also where you will edit your files (just like in normal Atom).
+- The bottom status bar shows you error and health statistics.
 
 ## Adding a Project
 
-After launching Nuclide, the next common step is to open a project that you would like to work on.
-This could be a Hack, Flow or any other project that has a root directory.
+The first common step after launching Nuclide is to open a project you would like to work on.
+This could be a Hack, Flow, or any other project that has a root directory.
 
-To add a project, you can click on the `Add Project Folder` button in the left-hand side pane, use
-the keyboard shortcut `Cmd-Shift-O` (`ctrl-shift-O` on Linux), or choose
-`File | Add Project Folder` from the Atom menu.
+To add a project, click the `Add Project Folder` button in the left side-pane, use
+the `Cmd-Shift-O` keyboard shortcut (`Ctrl-Shift-O` on Linux), or choose
+`File | Add Project Folder` from the Atom menu bar.
 
 ![](/static/images/docs/quick-start-getting-started-add-project.png)
 
-After adding a project you will see the root of your project at the top of the file tree and all
+After adding a project you will see the root of your project at the top of the File Tree with all
 files and folders as a tree hierarchy underneath it.
 
 ![](/static/images/docs/quick-start-getting-started-file-tree-view.png)
 
 ## Quick Launch Menu
 
-On the Nuclide home page you will find the *Quick Launch Menu* that gives quick access to many of
-the popular features of Nuclide. Click on the `Try It` button to use each feature.
+On the Nuclide Home page you will find the *Quick Launch Menu* that gives quick access to many of
+the popular features of Nuclide. Click the `Try It` button to use each feature.
 
 ![](/static/images/docs/quick-start-getting-started-quick-launch.png)
 
 ## Quick Open
 
-Nuclide's [*Quick Open*](/docs/features/quick-open) feature gives you access to Nuclide's file
-search mechanism, which includes *omnisearch* which can find anything in your opened project.
-Click on `Try It`, or `Cmd-T` (`ctrl-T` on Linux) to access the feature.
+The [*Quick Open*](/docs/features/quick-open) feature gives you access to Nuclide's file
+search mechanism, including *OmniSearch* which can find any files in your opened project and show you which have been recently opened.
```

Comment:
So, in theory we can find stuff in files.... but it depends on the type of project you are working on. Maybe we could say something like:

> , including *OmniSearch, which quickly displays recently opened files, quick searches for files based on partial names and, depending, on the project, can search within files for symbols, etc.

>